frontend/src/app/main/data/viewer.cljs

```diff
@@ -136,12 +136,18 @@
             (->> data
                  (filter #(= page-id (:page-id %)))
                  (d/index-by :id)
-                 (assoc state :comment-threads)))]
+                 (assoc state :comment-threads)))
+          (on-error [err]
+            (if (= :authorization (:type err))
+              (rx/empty)
+              (rx/throw err)))]
+
     (ptk/reify ::fetch-comment-threads
       ptk/WatchEvent
       (watch [_ state stream]
         (->> (rp/query :comment-threads {:file-id file-id})
-             (rx/map #(partial fetched %)))))))
+             (rx/map #(partial fetched %))
+             (rx/catch on-error))))))
```

Comment:
Now I can open view screen with an anonymous user. But It gives unauthorized error when I try to go to comments and code tabs.